index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
+  </head>
+  <body>
+    <div id="main">
+      <div id="title"><h1>Dr. Norman Borlaug</h1></div>
+      <div id="img-div">
+        <img id="image" src="./img/arch.jpg" alt="A Building" />
+        <div id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase
+          wheat yields - part of his life-long war on hunger.
+        </div>
+      </div>
```

Comment:
Can you use [a `figure` element here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure)? This would make it more semantically structured, note that using a `figure` element here also allows the use of `figcaption` which is better than the use of `div` here.